cloudbrain/rt_server/rt_server.py

```diff
@@ -86,29 +90,36 @@ def handle_channel_subscription(self, stream_configuration):
         device_id = stream_configuration['deviceId']
```

Comment:
The big comment block should be a docstring of `on_message`:

```
     # This will receive instructions from the client to change the
     # stream. After the connection is established we expect to receive a JSON
     # with deviceName, deviceId, metric; then we subscribe to RabbitMQ and
     # start streaming the data.
     #
     # NOTE: it's not possible to open multiple connections from the same client.
     #       so in case we need to stream different devices/metrics/etc. at the
     #       same time, we need to use a solution that is like the multiplexing
     #       in the sockjs-tornado examples folder.
```